app/models/workers_scaling/utils/logger.rb

```diff
@@ -0,0 +1,8 @@
+module WorkersScaling
+
+  LOGGER = Logger.new("#{Rails.root}/log/workers_scaling.log", 5, 1024*1024*100) unless defined? LOGGER
```

Comment:
Czy ten logger tworzy max. 5 plików po 100 MiB? Zmniejszcie może tak do max. 3 sztuk - jak ktoś sobie napisze skrypt archiwizujący to OK, ale jak ktoś uruchomi system bez takiej wiedzy to za jakiś czas dostanie 1 GiB logów.